.github/workflows/scripts/code_coverage_disable_check.py

```
@@ -97,7 +96,9 @@
             ):
                 if has_code_coverage_disable(item):
                     print(
-                        f"""File {item} contains code coverage disable statement. Please remove it and add the appropriate tests."""
+                        f"""\
+File {item} contains code coverage disable statement. \
+Please remove it and add the appropriate tests."""
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)